docs/guides/migration-from-2-to-3.md

```diff
@@ -1,7 +1,7 @@
 # Migration Guide for Enzyme v2.x to v3.x
 
 The change from Enzyme v2.x to v3.x is a more significant change than in previous major releases,
-due to the fact that the internal implementation has been almost completely rewritten.
+due to the fact that the internal implementation of Enzyme has been almost completely rewritten.
```

Comment:
`s/Enzyme/enzyme/g`?

Reply:
why? it's capitalized in other places?

Reply:
Historically we've not capitalized it anywhere; I think that lowercase (to match the repo and npm name) is a better choice.